docs/cloud-nativeness/docker-compose.md

```diff
@@ -45,18 +45,17 @@ If you change the Docker images in your Docker Compose generated file, ensure th
 the Gateway are built with the same Jina version to guarantee compatibility.
 ````
 
-## Example: Index and search images using CLIPEncoder and AnnLiteIndexer
+## Example: Index and search text using our own build Encoder and Indexer
```

Comment:
"our own build Encoder and Indexer" -- Unless it encodes and indexes builds, this is grammatically wrong, but I don't know exactly what it should be.

Reply:
own `built` sorry

Reply:
it is not Jina AIs, is the Executor that each user builds.

Reply:
If you mean "your own build" then just change it to that and I'll approve.